### PR TITLE
#4182 theming for mobile app

### PR DESCRIPTION
**Pull request by desfero**
A continuation of original #4192 

**Testing notes**
IOS: https://i.diawi.com/LKuZ8h
Android: https://i.diawi.com/wk7MgY
This pr:
- [ ] Adds tab navigation (https://github.com/Neufund/platform-frontend/issues/4205)
image
- [ ] Provides a proper app icon and app name for both android and ios

### CODE REVIEW

File: packages/wallet/app/themes/light/Light.ts [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import { EThemeVariant, ITheme } from "../types";
+import { lightColorSystem } from "./colorSystem";
+
+const navigationTheme = {
```
Comment (desfero) [inline packages/wallet/app/themes/light/Light.ts]: still need to connect theme with already created components, will do that soon in of the next pr with some component changes

File: packages/wallet/app/components/containers/AppContainer.tsx [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import React from "react";
+
+import { ThemeProvider } from "../../themes/ThemeProvider";
+
+const AppContainer: React.FunctionComponent = ({ children }) => (
```
Comment (arjunu) [inline packages/wallet/app/components/containers/AppContainer.tsx:5]: is this component going to get any more logic in the future?

File: packages/wallet/app/AppRouter.tsx [hunk 3]
```diff
@@ -1,25 +1,78 @@
...
-    </Stack.Navigator>
-  </NavigationContainer>
+type TBarProps = { color?: string };
+
+const Root: React.FunctionComponent = () => (
+  <Tab.Navigator initialRouteName={appRoutes.home}>
+    {tabConfig.map((route, index) => {
+      return (
+        <Tab.Screen
+          key={index}
```
Comment (arjunu) [inline packages/wallet/app/AppRouter.tsx:24]: why not use name as key or add a new id property?

File: packages/wallet/app/components/shared/Icon.tsx [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+import Wallet from "../../assets/wallet.svg";
+
+enum EIconType {
+  HOME = "home",
+  PORTFOLIO = "portf",
+  PROFILE = "profile",
+  WALLET = "wallet",
+}
+
+const getIcon = (type: EIconType) => {
```
Comment (arjunu) [inline packages/wallet/app/components/shared/Icon.tsx:16]: what if we have 30 icons? why not pass svg as children instead?

File: packages/wallet/app/components/containers/AppContainer.tsx (hunk 2 above)
Comment (desfero) [reply packages/wallet/app/components/containers/AppContainer.tsx:5]: I would say so, for eg. we will include `AppearanceProvider`  in the future here.

File: packages/wallet/app/AppRouter.tsx (hunk 3 above)
Comment (desfero) [reply packages/wallet/app/AppRouter.tsx:24]: good point, fixed to use route as it should be unique always

File: packages/wallet/app/components/shared/Icon.tsx (hunk 4 above)
Comment (desfero) [reply packages/wallet/app/components/shared/Icon.tsx:16]: discussed on slack

**Pull request merged**